client/stylesheets/_toCat.scss

```diff
@@ -5067,6 +5114,84 @@ On click the images will be displayed at normal size to complete the effect
 
 /* End */
 
+/* Header Location Radius */
+
+.headerLocationRadius {
+	float: left;
+	margin-left: 16px !important;
+	margin-top: 8px !important;
+	vertical-align: middle !important;
+	font-size: 0.9rem;
+}
+
+.headerRadius {
+	width: 70px;
+	margin-left: 10px;
+	padding: 8px 2px;
+	color: #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs